tools/wptrunner/docs/usage.rst

```diff
@@ -1,242 +0,0 @@
-
---------------------
-Running test subsets
---------------------
-
-To restrict a test run just to tests in a particular web-platform-tests
-subdirectory, specify the directory name in the positional arguments after
-the options; for example, run just the tests in the `dom` subdirectory::
-
-  wptrunner --metadata=~/testtwf/meta --tests=~/testtwf/web-platform-tests/ \
-    --binary=/path/to/firefox --certutil-binary=/path/to/certutil \
-    --prefs-root=/path/to/testing/profiles \
-    dom
-
--------------------
-Running in parallel
--------------------
-
-To speed up the testing process, use the ``--processes`` option to have
-wptrunner run multiple browser instances in parallel. For example, to
-have wptrunner attempt to run tests against with six browser instances
-in parallel, specify ``--processes=6``. But note that behaviour in this
-mode is necessarily less deterministic than with ``--processes=1`` (the
-default), so there may be more noise in the test results.
-
--------------------
-Using default paths
--------------------
-
-The (otherwise-required) ``--tests`` and ``--metadata`` command-line
-options/flags be omitted if any configuration file is found that
-contains a section specifying the ``tests`` and ``metadata`` keys.
-
-See the `Configuration File`_ section for more information about
-configuration files, including information about their expected
-locations.
-
-The content of the ``wptrunner.default.ini`` default configuration file
-makes wptrunner look for tests (that is, a web-platform-tests checkout)
-as a subdirectory of the current directory named ``tests``, and for
-metadata files in a subdirectory of the current directory named ``meta``.
-
-Output
-------
-
-wptrunner uses `the mozlog package
-<https://firefox-source-docs.mozilla.org/mozbase/mozlog.html>`_) for
-output. This structures events such as test results or log messages as
-JSON objects that can then be fed to other tools for
-interpretation. More details about the message format are given in
-`the mozlog documentation
-<https://firefox-source-docs.mozilla.org/mozbase/mozlog.html>`_.
-
-By default the raw JSON messages are dumped to stdout. This is
-convenient for piping into other tools, but not ideal for humans
-reading the output. `mozlog
-<https://firefox-source-docs.mozilla.org/mozbase/mozlog.html>`_ comes
-with several other formatters, which are accessible through command
-line options. The general format of these options is
-``--log-name=dest``, where ``name`` is the name of the format and
-``dest`` is a path to a destination file, or ``-`` for stdout. The raw
-JSON data is written by the ``raw`` formatter so, the default setup
-corresponds to ``--log-raw=-``.
-
-A reasonable output format for humans is provided as ``mach``. So in
-order to output the full raw log to a file and a human-readable
-summary to stdout, one might pass the options::
-
-  --log-raw=output.log --log-mach=-
-
-Configuration File
```

Comment:
We should have some documentation of the config file format somewhere. Because I can never remember what the format is, even though it's rarely used aside from by vendors (cc @jgraham).